cli/cargo-ohrs/src/cli/mod.rs

```diff
@@ -0,0 +1,71 @@
+mod artifact;
+mod build;
+mod cargo;
+mod init;
+mod publish;
+
+use bpaf::{construct, Doc, OptionParser, Parser};
+use owo_colors::colors::CustomColor;
+use owo_colors::OwoColorize;
+
+use artifact::cli_artifact;
+use build::cli_build;
+use cargo::cli_cargo;
+use init::cli_init;
+use publish::cli_publish;
+
+pub fn cli_run() -> OptionParser<crate::Options> {
+  let init = cli_init()
+    .to_options()
+    .command("init")
+    .help("Project initialization");
+  let build = cli_build()
+    .to_options()
+    .command("build")
+    .help("Project build");
+  let artifact = cli_artifact()
+    .to_options()
+    .command("artifact")
+    .help("Generate ohpm .har file");
+  let publish = cli_publish()
+    .to_options()
+    .command("publish")
+    .help("Publish ohpm package, but not implement yet.");
+  let cargo = cli_cargo()
+    .to_options()
+    .command("cargo")
+    .help("Used to execute any cargo command and ensure it is under the ohpm environment.");
+
+  construct!([init, build, artifact, publish, cargo]).to_options()
```

Comment:
Can we add version options for subcommand?  @pacak

Reply:
Any `.to_options()` gives you some `OptionParser<T>` back that implements `.version()` amongst other things, so something like this is perfectly valid

```rust
  let init = cli_init()
    .to_options()
    .version("1.0")
    .command("init")
    .help("Project initialization");
```

You can even have different versions for different subcommands.